lsp-mode.el

```diff
@@ -5007,6 +5007,12 @@ In addition, each can have property:
   :group 'lsp-mode
   :type 'boolean)
 
+(defcustom lsp-enable-server-download t
```

Comment:
Maybe put "auto" in the name somewhere? Since this doesn't affect the ability to use e.g. `lsp-install-server`

Reply:
Make sense to me WDYT @yyoncho ?

Reply:
I added the word suggest, because the "auto" is not actually auto.